src/Forms/FormField.php

```diff
@@ -951,16 +948,12 @@ public function Field($properties = [])
             $context = $context->customise($properties);
         }
 
-        $result = $context->renderWith($context->getTemplates());
+        $dbHtmlText = $context->renderWith($context->getTemplates());
```

Comment:
I personally prefer `$result` here - variables generally indicate what a value will be used for, now what type it is. That's subjective though so I won't hold back merging for this.

Reply:
Keep as is